### PR TITLE
Apicompliance/617 Elasticexplore changes

### PR DESCRIPTION
**Pull request by alfinhaji**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

Updated both elasticexplore action API from limit=30 to limit=24 to reflect the current statue in QCS.  Also updated the API call change when tag drop down is selected (only seen when the tag parameter is provided in scenario and thus filter by collectionID)

### CODE REVIEW

**Comment by alfinhaji** [comment]
Tested both qlik-oss and internal versions, changes succeed and pass, traffic logs reflect correct changes.

File: docs/settingup.md [hunk 1]
```diff
@@ -1602,7 +1602,7 @@ Explore the hub for apps and fill the artifact map with apps to be used by other
 ### Settings
 
 * `keepcurrent`: Keep the current artifact map and add the results from the `elasticexplore` action. Defaults to `false` (that is, empty the artifact map before adding the results from the `elasticexplore` action), if omitted.
-* `paging`: Go through all app pages in the hub. Defaults to `false` (that is, only include the first 30 apps that the user can see), if omitted.
+* `paging`: Go through all app pages in the hub. Defaults to `false` (that is, only include the first 24 apps that the user can see), if omitted.
```
Comment (DnlLrssn) [inline docs/settingup.md:1605]: you're not supposed to edit this file directly it's generated. You can find out how generating documentation works here:
https://github.com/qlik-oss/gopherciser/tree/master/generatedocs
Basically you need to edit the `params.json` file then run `go generate`

Comment (alfinhaji) [reply docs/settingup.md:1605]: Ok sounds good thanks!

Comment (alfinhaji) [reply docs/settingup.md]: I will look at updating the doc in a separate PR

Comment (alfinhaji) [reply docs/settingup.md:1605]: OK done!

**Pull request merged**